lib/rules/length-zero-no-unit/__tests__/index.js

```diff
@@ -12,6 +12,18 @@ testRule(rule, {
       code: "a { top: 0; }",
       description: "unitless zero"
     },
+    {
+      description: "ignore calc",
+      code: "a { padding: calc(0px +\n 0px); }"
+    },
+    {
+      description: "ignore calc. insensitive",
+      code: "a { padding: cAlc(0px + 0px); }"
+    },
+    {
+      description: "ignore calc. empty calc",
+      code: "a { padding: calc(); }"
+    },
```

Comment:
Can we add test like `padding: calc(1in + 0in * 2)) 0px`, should have one error

Reply:
awesome suggestion. I have improve it to `padding: calc(1in + 0in * 2)) 0in calc(0px) 0px` since it was needed to rework regexp, too